liota/entities/devices/device.py

```diff
@@ -59,11 +59,11 @@ def __init__(self, name, parent, entity_id, entity_type="Device"):
             entity_type=entity_type
         )
 
-    def register(self, entity_obj, dcc_obj, reg_entity_id):
+    def register(self, dcc_obj, reg_entity_id):
```

Comment:
Register is an abstract method in the base class will have to be overridden in the child class and better to keep the implementation in device.py as the register implementation might differ from system.